content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Configure_the_Primary_YubiHSM_2_Device.adoc

```diff
@@ -101,130 +115,82 @@ One
 
-*Step 8*: When the relevant prompt is displayed, each of the three wrap key custodians should take their turn in front of the screen to record their share. A warning notice appears advising you/them  that the shares are not stored anywhere. It is important that each custodian record the whole string presented, including the prefix (in this example, `2-1-`) which indicates the number of shares required to regenerate the key (or the privacy threshold) and the number identifying where in the sequence the share was created.
+The wrap key thereby generated is saved to the HSM 2 device.
 
-a) To start having the custodians record the key shares, press **Enter**.
-b) The first custodian records his or her share and confirms by pressing `y` that the share was recorded.  The screen buffer is cleared before the next share is presented.
-c) The next custodian records the key share for the second share, confirms it, and so on.
+*Step 8*: When the relevant prompt is displayed, each of the three wrap key custodians should take their turn in front of the screen to record their share.
+
+[IMPORTANT]
+============
+Each custodian must record the whole string presented, including the prefix (in the following example, `2-1-`) which indicates the number of shares required to regenerate the key (the privacy threshold) and the number identifying where in the sequence the share was created.
+============
 
-Following is an example of a share presented on the screen:
+The following is an example of a share presented on the screen:
 
 ....
 2-1-WWmTQj5PHGJQ4H9Y2ouURm8m75QkDOeYzFzOX1VyMpAOeF3YKYZyA…
 Have you recorded the key share? (y/n)
 ....
 
-*Step 9*: The setup configuration continues by asking if you want to create an **application authentication key**. This key is used to authenticate to the device by the KSP in Windows, allowing the KSP to perform operations in YubiHSM 2. Since object IDs `1` and `2` are already in use by the default authentication key and the wrap key respectively, the example in this guide assumes that the application authentication key to be created gets ID `3`. Do one of the following:
+A notice is displayed, warning that the shares are not stored anywhere.
 
-* To manually assign an application authentication key ID, type `3`.
-* To allow the system to generate a wrap key ID automatically, type `0`.
-
-*Step 10*: You also need to choose a password for the application authentication key. Be sure to store the password of the application authentication key so that it cannot be compromised. You will need this information later to configure the YubiHSM KSP DLL. The password should be at least eight (8) characters. Enter the application authentication key password. A confirmation message appears:
-
-....
-Stored application authentication key with ID 0x0003 on the device
-Saved wrapped application authentication key to 0x0003.yhw
-....
+a) To start having the custodians record the key shares, press **Enter**.
```

Comment:
minor thing: if we switch this to a. instead of a) (and later items in the list to b. c. etc) it will build a "real" list out of it. This should obviously be done consistently if we want it rendered like so.

Reply:
Thanks, have fixed the two instances of "lettered numbering" in this file, and will check for others.